Rakefile

```diff
@@ -90,9 +90,6 @@ namespace :admin do
       puts 'deleting analytics buckets'
       Cluster::S3AnalyticsBuckets.delete
 
-      puts 'deleting cloudwatch log groups'
-      Cluster::CWLogs.delete
-
```

Comment:
Just curious, why is delete cw logs removed

Reply:
Added a `Cluster::Layers.delete_log_groups` step to `admin:cluster:delete`